src/pages/es/core-concepts/partial-hydration.md

```diff
@@ -7,7 +7,7 @@ setup: |
 i18nReady: true
 ---
 
-**Astro por defecto genera todas las páginas web sin JavaScript del lado del cliente.** Use cualquier componente de interfaz de usuario que desee ([React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) y [Lit](https://lit.dev/)), Astro lo renderizará automáticamente a HTML en la compilación final y eliminará todo el JavaScript. Esto mantiene cada página más rápida por defecto.
+**Astro por defecto genera todas las páginas web sin JavaScript del lado del cliente.** Use cualquier componente de interfaz de usuario como ([React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) o [Lit](https://lit.dev/)), Astro lo renderizará automáticamente a HTML en la compilación final y eliminará todo el JavaScript. Esto mantiene cada página más rápida por defecto.
```

Comment:
Thanks, @kevinzunigacuellar !

The intention for the change here was to remove "ANY component" and make it "A component."" (un componente ?)

The feedback we received was, "Can I use ANY kind of component, or only the ones on this list?" So we need the change to reflect "Use a component from React, Preact... etc. and Astro will..."

Reply:
now the Spanish version sounds more like "use any of the following UI frameworks" because using "a" sounds like you can only use one framework.